Name:  app5_2_1000_7_ Lr_rate:  1e-07 Use Teacher:  False  Riter:  10  Subjs:  2
Using device: cuda:0
['Brats17_CBICA_BFB_1_t2_unbiased.nii.gz']
validation subject ['Brats17_CBICA_BHK_1_t2_unbiased.nii.gz']
Loading train set for subj
Subject  Brats17_CBICA_BFB_1_t2_unbiased.nii.gz  Number of Slices:  143
Traceback (most recent call last):
  File "train_restore_MAP_NN.py", line 130, in <module>
    device, writer, seg, mask, log=bool(batch_idx%2))
  File "/scratch_net/biwidl214/jonatank/code_home/restor_MAP/restoration.py", line 146, in train_run_map_NN
    loss.backward()#retain_graph=True)
  File "/scratch_net/biwidl214/jonatank/anaconda3/envs/JKMT/lib/python3.7/site-packages/torch/tensor.py", line 195, in backward
    torch.autograd.backward(self, gradient, retain_graph, create_graph)
  File "/scratch_net/biwidl214/jonatank/anaconda3/envs/JKMT/lib/python3.7/site-packages/torch/autograd/__init__.py", line 99, in backward
    allow_unreachable=True)  # allow_unreachable flag
RuntimeError: CUDA out of memory. Tried to allocate 128.00 MiB (GPU 0; 11.91 GiB total capacity; 11.24 GiB already allocated; 59.12 MiB free; 11.32 GiB reserved in total by PyTorch)
